qingstor/sdk/client/upload_client.py

```diff
@@ -0,0 +1,82 @@
+from qingstor.sdk.service.qingstor import QingStor
+from file_chunk import FileChunk
```

Comment:
Follow the import style in https://github.com/yunify/qingstor-sdk-python/blob/master/qingstor/sdk/request.py